packages/cds-annotation-parser/package.json

```
@@ -0,0 +1,64 @@
+{
+    "name": "@sap-ux/cds-annotation-parser",
+    "version": "0.1.0",
+    "description": "SAP Fiori tools - Annotation Parser for CDS",
+    "publisher": "SAPSE",
+    "author": "SAP SE",
+    "license": "Apache-2.0",
+    "main": "dist/index.js",
+    "scripts": {
+        "build": "npm-run-all clean compile",
+        "compile": "tsc --build ./tsconfig.build.json",
+        "clean:dist": "rimraf ./dist ./generators *.tsbuildinfo",
```

There is no generators folder here.
```suggestion
        "clean:dist": "rimraf ./dist *.tsbuildinfo",
```

---

Removed